pkg/api/image/formatter.go

```diff
@@ -127,7 +187,7 @@ func (h UploadActionHandler) uploadImage(rw http.ResponseWriter, req *http.Reque
 	return nil
 }
 
-func (h UploadActionHandler) waitForBackingImageDataSourceReady(name string) error {
+func (h ImageHandler) waitForBackingImageDataSourceReady(name string) error {
 	retry := 30
```

Comment:
not sure `60s` is enough for slow environment. Is it marked in Harvester Doc ?

Reply:
@w13915984028 The value here is not added in this PR. Please create another issue if you feel it's not reasonable in the image upload path.

Reply:
Thanks, @bk201, 
Agree. that 30s is the previous commits mechanism.
I thought that if we want a more timeout, we could raise another issue to discuss what value is more suitable.

Reply:
Let's track it in https://github.com/harvester/harvester/issues/2718 .